backend/api_calls.py

```diff
@@ -261,17 +261,21 @@ def AddGroup(request, firebase):
   """
   results = []
   valid_args = ['!groupId', 'gameId', 'ownerPlayerId', 'allegianceFilter']
```

Comment:
To avoid duplication, you could just append ownerPlayerId to valid_args once you've created required_args.